starknet_devnet/blueprints/postman.py

```diff
@@ -55,3 +57,30 @@ async def flush():
 
     result_dict = await state.starknet_wrapper.postman_flush()
     return jsonify(result_dict)
+
+
+@postman.route("/l1_to_l2", methods=["POST"])
+async def l1_to_l2():
+    """L1 to L2 message mock endpoint"""
+    request_json = request.json or {}
+
+    # Generate transactions
+    transaction = InternalL1Handler.create(
```

Comment:
Look at other transaction endpoints (in gateway.py) and how they rely on `validate_transaction`, and convert to internal counterpart only inside the StarknetWrapper method.

Reply:
discussed on call